### ISSUE TITLE
PRO: filterset_class in ListModelView that is used by ModelViewSet is being overriden

### ISSUE THREAD

**Issue by emihir0** [viewflow/django-material#363]
With ModelViewSet we can use our own view classes, such as some ListModelView class; where we, for example, want to specify extra columns in list_display so the main ModelViewSet does not get bloated and is easy to navigate. The problem is that if you specify `filterset_class` in ListModelView, such as:

```
class MyListView(ListModelView):
    model = models.MyModel
    list_display = [...]
    filterset_class = filters.MyModelFilter
```

And then use this listview in a ModelViewSet, such as:

```
class MyModelViewSet(ModelViewSet):
    model = models.MyModel
    
    list_view_class = MyListView
```

the `filterset_class` of `MyListView` will be ignored. This happens because in `filter_kwargs` of `ModelView`:

https://github.com/viewflow/django-material/blob/master/material/frontend/views/viewset.py#L132-L134

we check against whether the `filterset_class` is `DEFAULT` (`object()`), which it is not by 'default', it is simply None (line 130 & 131 of DJANGO-MATERIAL PRO). It should be an easy fix, simply change the `None` to `DEFAULT`.

TL:DR: `filterset_class` does not work in when `ModelViewSet` points to a simple `ListModelView`

**Comment by kmmbvnr** [comment]
Fixed in django-material-pro==1.1.2
